manifests/init.pp

```diff
@@ -102,9 +105,9 @@
   $use_sympa           = false,         # postfix_use_sympa
   $postfix_ensure      = 'present',
   $mailx_ensure        = 'present',
-) inherits postfix::params {
-
+) { include ::postfix::params
```

Comment:
While I agree inheritance is not required here, I don't think it's in the scope of this PR to change it.